design/src/rfcs/rfc0027_improve_sdk_request_id_access.md

```diff
@@ -0,0 +1,244 @@
+RFC: Improving access to request IDs in SDK clients
+===================================================
+
+> Status: RFC
```

Comment:
We can update the status to something like

Implemented in [smithy-rs#2129](https://github.com/awslabs/smithy-rs/pull/2129)